CI/appveyor/build.sh.inc

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Options
+MAKE_J=-j8
```

Comment:
this is still hard-coded here

Reply:
I misunderstood things. Will remove, thanks!